pkgs/development/python-modules/importlab/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, networkx
+, python27
+}:
+
+buildPythonPackage {
+  pname = "importlab";
+  version = "0.7";
+
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "importlab";
+    rev = "676d17cd41ac68de6ebb48fb71780ad6110c4ae3";
+    sha256 = "sha256-O8y1c65NQ+19BnGnUnWrA0jYUqF+726CFAcWzHFOiHE=";
+  };
+
+  propagatedBuildInputs = [ networkx ];
+
+  checkInputs = [ python27 ];
```

Comment:
importlab requires Python2.7 to run tests.

Reply:
Can the tests which requires Python 2 be excluded?

Reply:
@fabaff 
Yes, I excluded `tests/test_parsepy.py` which requires Python 2.